### DISCUSSION TITLE
Separate SDK docs from the main PocketBase docs.

### DISCUSSION THREAD

**Discussion by BanDroid** [pocketbase/pocketbase#6167] [Q&A]
the current docs always showing code block of javascript and dart sdk, this is not a good practice for public documentation of pocketbase which could be used by many frameworks other than javascript and dart.

i hope the code blocks would show more example of using plain rest api features like fetch in javascript or requests in python or retrofit in java, or whatever, as long as it could be used by other languages.

the current docs looks like forcing people to use the sdk and not a very beginners-friendly.

**Comment by ganigeorgiev** [comment]
Whether this is a "good practice" or "not a very beginners-friendly" is subjective and I personally disagree with that statement. For the time being I don't plan adding new SDKs or other examples because I simply don't consider it necessary, nor I have the time for that (make sure to read [FAQ](https://pocketbase.io/faq/)).

In any case there is no question here.

If you want something to change in regards to the docs, feel free to open a PR to https://github.com/pocketbase/site with your suggestion.

If you have an issue with some of the API usage, elaborate what you are trying to do and I'll try to assist, but as of now this reads as a complain.

**Reply by BanDroid** [reply]
sorry for misunderstanding, it is not a complain, more like a suggestion.

for example, the realtime api is very hard to accomplish without the sdk. so working with realtime data (e.g android java), i have to use a trick which is using hidden webview and load a javascript from it, and send data back and forth, need to rely on sdk.

hopefully there's more clear explanation on accomplish it without the sdk, like through plain websocket in the docs.
